File: src/main/scala/Models/EstudanteCesarSchool.scala
package Models
import Traits.{Desesperado, Lascado}

class EstudanteCesarSchool(nome: String, idade: Int) extends Mamifero(nome, idade) with Lascado with Desesperado{

  override val isLascado: Boolean = true
  override val isDesesperado: Boolean = true

  override def sound(): String = "joao fudeu meu find"

}

File: Java/src/models/EstudanteCesarSchool.java
package models;

import interfaces.Desesperado;
import interfaces.Lascado;

public class EstudanteCesarSchool extends Mamifero implements Lascado, Desesperado {
	
	public EstudanteCesarSchool(String nome, int idade) {
		super(nome, idade);
	}

	@Override
	public String sound() {
		return "to cansado";
	}

	@Override
	public boolean isDesesperado() {
		return true;
	}

	@Override
	public boolean isLascado() {
		return true;
	}

}
